src/js/comments.js

```diff
@@ -0,0 +1,22 @@
+import * as API from './api.js';
+
+const displayCommentsPopup = async ({ mealId }) => {
+  const meal = await API.getMealById(mealId);
+  const modal = document.querySelector('.modal');
```

Comment:
Place the content inside the `modal-content` instead of `modal`. E.g
`const modal = document.getElementById('modal-content')`